### PR TITLE
Todo list

### PR DESCRIPTION
**Pull request by kubilaycaglayan**
Dear TSE,

In this project, we have built a todo list application using JavaScript, Webpack and Bootstrap, following the requirements of Microverse and The Odin Project. Our project includes the following features:
- We have created two factory functions to create Projects and Todos.
- We have created different modules in different files to host the DOM, the storage, the helper functions and the events on our website.
- We have created a straight-forward IU for the users.
- We have designed the website using Bootstrap.
- We have used Webpack to join all our modules and functions together.
- We have deployed the website using Github Pages.
- User can create projects and todos inside every single project.
- The user finds a default project when first lands on the website.
- User can delete projects and todos.
- User can edit todos.
- The todos are highlighted with different colours depending on the priority of each todo.

Best Regards

### CODE REVIEW

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @kubilaycaglayan , 

Great job

### [required changes]

You need to do form controls to make sure empty forms cannot be submitted
I think that's basically what needs to be done
todo

todo2

Happy coding

**Comment by javitocor** [comment]
We have added validation for the forms as requested.

File: src/models/events.js [hunk 1]
```diff
@@ -17,13 +17,17 @@ import {
...
+    if (nameProject !== '') {
+      const ourStore = storage();
+      const newProject = createProject(nameProject);
+      ourStore.addProject(newProject);
+      const closeModalButton = document.getElementById('closeModalButton1');
+      closeModalButton.click();
+      document.getElementById('projectForm').reset();
+      dom().showProject();
+    } else {
+      window.alert('You need to fill the input field!');
```
Comment (bellom) [inline src/models/events.js]: I trust you can do this 👇  before merging, `required` is already doing the job
`This should be production code, we shouldn't have `alert`, please remove the `alert`.`

My 👀 on YOU 😃

File: src/models/events.js [hunk 2]
```diff
@@ -38,14 +42,18 @@ const events = function events() {
...
+      const newTodo = createToDoItem(todoValues[0], todoValues[1], todoValues[2], todoValues[3], todoValues[4]);
+      const projectId = document.getElementById('projectId').innerHTML;
+      const ourStore = storage();
+      ourStore.addTodo(projectId, newTodo);
+      const closeModalButton = document.getElementById('closeModalButton');
+      closeModalButton.click();
+      document.getElementById('todoForm').reset();
+      dom().showTodos(projectId);
+    } else {
+      window.alert('You missed fill up one or more fields!');
```
Comment (bellom) [inline src/models/events.js]: I trust you can do this 👇  before merging, `required` is already doing the job
`This should be production code, we shouldn't have `alert`, please remove the `alert`.`

My 👀 on YOU 😃

**Review by bellom — APPROVED** [review]
## Hi Team :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**